lib/services/ngsi/entities-NGSI-v2.js

```diff
@@ -28,23 +28,21 @@
 /* eslint-disable consistent-return */
```

Comment:
Maybe we should take this opportunity to ensure that all function in the file is properly documented in Javadoc. In particular:

* Review existing Javadocs, to ensure they are correct (may be obsolete)
* Add Javadocs to function that doesn't have it yet (e.g. `function addTimestampNgsi2`)